From: Casey Bodley <cbodley@redhat.com>
Date: Tue, 16 Aug 2016 14:41:14 -0400
Subject: rgw: RGWCoroutinesManager::run returns status of last cr

Fixes: http://tracker.ceph.com/issues/17047

Signed-off-by: Casey Bodley <cbodley@redhat.com>
(cherry picked from commit 389ff16f4b84f06d2df15af216a99c9f2ea1552b)
Resolves: rhbz#1368562
---
 src/rgw/rgw_coroutine.cc | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/rgw/rgw_coroutine.cc b/src/rgw/rgw_coroutine.cc
index 455c178..f9a64d1 100644
--- a/src/rgw/rgw_coroutine.cc
+++ b/src/rgw/rgw_coroutine.cc
@@ -574,8 +574,6 @@ int RGWCoroutinesManager::run(list<RGWCoroutinesStack *>& stacks)
     if (iter == scheduled_stacks.end()) {
       iter = scheduled_stacks.begin();
     }
-
-    ret = 0;
   }
 
   lock.get_write();
